cache/cache.go

```diff
@@ -0,0 +1,8 @@
+package cache
+
+import "github.com/daniyakubov/book_service_n/models"
+
+type Cache interface {
```

Comment:
WDYT on `ActivityCacher` name (same comment as before for the book store interface) 
and you can create a pkg that will contain all the interfaces of the datastores at the same place